modules/tinymce/changelog.txt

```diff
@@ -18,6 +18,8 @@ Version 5.7.0 (TBD)
     Fixed the "Dropped file type is not supported" notification incorrectly showing when using an inline editor #TINY-6834
     Fixed an issue with external styles bleeding into TinyMCE #TINY-6735
     Fixed incorrect return types on `editor.selection.moveToBookmark` #TINY-6504
+    Fixed the type signature for `editor.selection.getRng` incorrectly returning null #TINY-6843
```

Comment:
This needs to be `Changed` not `Fixed`, as it was intentional that happened previously.

- [ ] Change this to `Changed` and move to the appropriate position